src/v2/guide/list.md

```diff
@@ -415,17 +415,17 @@ Vue.set(example1.items, indexOfItem, newValue)
 example1.items.splice(indexOfItem, 1, newValue)
 ```
 
-To deal with caveat 2, you can also use `splice`:
+Pour gérer la seconde limitation, vous pouvez également utiliser `splice` :
 
 ``` js
 example1.items.splice(newLength)
 ```
 
-## Displaying Filtered/Sorted Results
+## Affichage de résultats filtrés/triés
```

Comment:
les deux traductions sont valables, en anglais de nombreux verbes ne font pas la différence entre le participe présent et le nom associé (exemple: the growing / growing). On peut garder la tienne, c'est  une question de préférences.